index.html

```diff
@@ -127,7 +128,259 @@ <h2>Multi-Post Stories</h2>
                                 </ul>
                             </li>
                         </ul>
-                        <button type="button">See Project</button>
+                        <button type="button" class="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="myProjects">
+            <div class="project">
+                <div class="projectHeader">
+                    <h2>Profesional Art Printing Data</h2>
+                </div>
+
+                <div class="projectBody">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="projectHTML">
+                            <ul>
+                                <li>html</li>
+                            </ul>
+                        </li>
+                        <li class="projectBootstrap">
+                            <ul>
+                                <li>bootstrap</li>
+                            </ul>
+                        </li>
+                        <li class="projectRuby">
+                            <ul>
+                                <li>ruby</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <div class="projectButton">
+                    <button type="button">See Project</button>
```

Comment:
- [ ] Please make sure to follow the button styling: Active, Hover and Pressed, that's shown in the Figma template
![image](https://user-images.githubusercontent.com/28109626/148608366-95875c69-82cb-4e5d-adfb-f832c79667b1.png)

Reply:
Hello @mike2611 Thank you for this review.
- I'm sorry but from the figma I cannot find the color code for when pressed. Kindly assist me with this.

Reply:
Hello @Nemwel-Boniface, I see you have made the required changes. But let me tell you how you can look into any color code you need using your web browser developer tools. You can access your developer tools in Chrome by pressing `ctrl+shift+i`. There you have a tool called color picker using this tool you can get any code for any color that's shown in your web browser.

![image](https://user-images.githubusercontent.com/28109626/148660493-9537caf7-a45a-446d-bade-fe482328de84.png)

Reply:
Hello @mike2611 thank you for this tip!